packages/types/definitions.ts

```diff
@@ -197,6 +197,39 @@ export type HttpConfig = {
    * @title Extended metrics
    */
   extendedMetrics?: boolean;
+
+  /**
+   * https://www.artillery.io/docs/reference/engines/http#default-configuration
+   * @title Default HTTP engine configuration
+   */
+  defaults?: {
+    /**
+     * Default headers to be used in all requests.
+     * @title Headers
+     */
+    headers?: object;
+    /**
+     * Default cookies to be used in all requests.
+     * @title Cookie
+     */
+    cookie?: object;
+    /**
+     * Whether to turn on strict capture by default
+     * for all captures.
+     * https://www.artillery.io/docs/reference/engines/http#turn-off-strict-capture
+     * @default false
+     * @title Strict capture
+     */
+    strictCapture?: boolean;
```

Comment:
I think it might be possible for this to be a string due to being able to override with environment variables

Reply:
Great point. Will annotate this as a string union for now. We may consider casting all plain literal values of the definition to `X | string` at some point to satisfy the YAML format intricacies.